packages/loopring_v3.1/contracts/impl/modules/helpers/WithdrawalModule.sol

```diff
@@ -0,0 +1,210 @@
+/*
+
+  Copyright 2017 Loopring Project Ltd (Loopring Foundation).
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+
+
+import "../../../iface/modules/helpers/IWithdrawalModule.sol";
+import "./OnchainRequestModule.sol";
+
+import "../../../iface/IExchangeV3.sol";
+import "../../../lib/AddressUtil.sol";
+import "../../../lib/MathUint.sol";
+
+
+/// @title WithdrawalModule
+/// @author Brecht Devos - <brecht@loopring.org>
+contract WithdrawalModule is OnchainRequestModule, IWithdrawalModule
```

Comment:
How about move this to its parent directory and
rename  OnchainRequestModule to AbstractOnchainRequestModule
rename WithdrawalModule to AbstractWithdrawalModule